Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM php:7.3-fpm-alpine
+RUN apk add --update alpine-sdk python
+RUN git clone --recursive https://github.com/adsr/phpspy.git
+RUN cd phpspy && make
```

Comment:
just add
```
FROM php:7.3-fpm-alpine 
COPY --from builder /phpspy/phpspy /bin/phpspy
```

and you will much small image size

Reply:
This needs to be `--from=builder`, for anyone trying to replicate these steps